packages/dingus/package.json

```diff
@@ -8,7 +8,11 @@
     "markdown-it",
     "commonmark"
   ],
-  "repository": "accordproject/markdown-transform",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/accordproject/markdown-transform.git",
+    "directory": "packages/dingus"
+  },
```

Comment:
I don't believe we want to worry about the dingus in this, @jeromesimeon thoughts?

Reply:
So should I revert changes for this file?

Reply:
I think it's fine to leave it - no harm.